docs/module_web_with_python.md

```diff
@@ -5,137 +5,437 @@
 #### How the “ternary expression” looks like in Python?
+* return 'a' if (c > b) else 'f' <-- This returns 'a' if c is bigger than b, else it return 'f'.
 #### How the ternary expression looks like in JavaScript?
+* return c > b ? 'a' : 'f' <-- This returns 'a' if c is bigger than b, else it return 'f'.
 #### How to import a function from another module in Python?
+* 'from [module] import [function] as [rename]' <-- Imports a single function from a module and renames it.
+* 'import [module]' <-- Imports all functions from a module.
+* 'from [module] import [function1], [function2]' <-- Import multiple functions from a single module.
+* 'import [module], [module]' <-- Import all functions from multiple modules
 #### How to import a function from another module in JavaScript?
+* import defaultExport from "module-name"; <-- imports the defaultExport from a module.
+* import { export1 } from "module-name"; <--Import a single export from a module.
+* import { export1 as alias1 } from "module-name"; <-- Import a single export from a module and rename it.
+* import { export1 , export2 } from "module-name"; <-- Import multiple exports from a single module.
 
 ### Functional
 #### What is recursion?
+* A recursive  function is when a function calls itself again based on a set of conditions.
 #### Write a recursive function which calculates the Fibonacci numbers!
+*	def fib(n):
+    	    if n == 1 or n == 2:
+                return 1
+    	    return fib(n-1)+fib(n-2)
+        print(fib(15))
 #### How to store a function in a variable in Python?
+* x = def foo(): print('asd')
 #### List the ways of defining a callable logical unit in JavaScript!
+* function squareNumbers(number){return number * number}
+* squareNumbers: funciton(number) {return number * number}
+* squareNumbers = (number) => number**2
 #### What is an event listener? How to attach one?
+* Event listeners attach a Javascript function to DOM elements that are only called when a specified 'event' is happening.
+* Such events include 'click', 'dragstart', 'dragover', 'dragend'.
+* Example:
+	let x = document.querySelector('#id')
+	x.addEventListener('click', function(event){ body of the function })
 #### How to trigger an event in JavaScript?
+* The first argument in the event listener hints to the method of how to trigger it's function.
+* 'click' <-- function triggers upon clicking at the target,
+* 'dragstart' <-- function triggers upon starting to drag the object,
+* 'dragover' <-- function triggers upon dragging something over the event target.
+* 'dragend' <-- function triggers upon deciding to end the dragging of the event target,
+* 'drop' <-- function triggers upon dropping the event target to somewhere other than where it started from originally.
 #### What is a callback function? Tell some examples of its usage.
+* Callback are when a function is used as argument in another function.
```

Comment:
Typo:
Version a: "Callback" -> "Callbacks"
Version b: "Callback are" -> "Callback is"